src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -114,6 +114,9 @@ private async Task StartWorkerAsync()
             var client = await RemoteHostClient.TryGetClientAsync(_workspace, cancellationToken).ConfigureAwait(false);
             if (client == null)
             {
+                // Now that we've started, let the VS todo list know to start listening to us
+                _eventListenerTracker.EnsureEventListener(_workspace, this);
```

Comment:
can we just pull this above the call to TryGetClient, and also remove the EnsureEventListener below.  The goal of this was simply to wait until StartWorkerAsync, not necessarily to wait until the remote connection was made.

we could also just do it in StartAsync.  either woudl be fine.  

--

i would very much prefer not to do it in two places.

Reply:
➡️ Done